podman/domain/containers_create.py

```diff
@@ -422,6 +422,9 @@ def to_bytes(size: Union[int, str, None]) -> Union[int, None]:
             "work_dir": pop("working_dir"),
         }
 
+        for device in args.pop("devices", []):
+            params["devices"].append({"path": device})
```

Comment:
curious if we want to add other fields supported by the API... though this seems good enough for a first implementation.

Reply:
For now I just wanted to match to [the previously written help for devices](https://github.com/containers/podman-py/blob/main/podman/domain/containers_create.py#L56-L61). To be fair I'm not so sure whether the [rest of the parameters in API](https://github.com/containers/podman-py/blob/main/podman/domain/containers_create.py#L56-L61) works properly, especially taking into account [these comments](https://github.com/containers/podman-py/blob/main/podman/domain/containers_create.py#L293-L300).
So I've decided that unblocking passing devices is good for now, up until the `Major/Minor` is figured out